app/controllers/comments_controller.rb

```diff
@@ -1,6 +1,7 @@
 class CommentsController < ApplicationController
   before_action :set_comment, only: %i[ show edit update destroy ]
   before_action :is_an_authorized_user, only: [:destroy, :create]
```

Comment:
Line 3 is redundant if you are using before_action { authorize @comment || Comment}